Content.Server/ReagentOnItem/EntitySystems/SpaceLubeOnItemSystem.cs

```diff
@@ -0,0 +1,68 @@
+using Content.Shared.IdentityManagement;
+using Content.Shared.Popups;
+using Content.Shared.ReagentOnItem;
+using Content.Shared.Throwing;
+using Robust.Shared.Containers;
+using Robust.Shared.Random;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Inventory;
+using Content.Shared.Examine;
+
+namespace Content.Server.ReagentOnItem;
+
+public sealed class SpaceLubeOnItemSystem : EntitySystem
+{
+    [Dependency] private readonly ThrowingSystem _throwing = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+    [Dependency] private readonly InventorySystem _inventory = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SpaceLubeOnItemComponent, ContainerGettingInsertedAttemptEvent>(OnHandPickUp);
```

Comment:
Is this supposed to be on any container insertion or only on hands pickup?

Reply:
I honestly don't really know for this one! This is what it was before and I kind of assumed it was chosen for a reason but its very odd because if you try to inspect the item it will trigger. Do you think I should change it?

Reply:
(Vid showing)

https://github.com/space-wizards/space-station-14/assets/107373427/1245cbf5-dbb5-401c-a5aa-4af6dcc400d6

Reply:
The case I was thinking of was that if you already have the item in your hand when it becomes slippery, it'll run this code when you try to put it in a backpack.

Maybe try that and see if it does anything. If it doesn't do anything, maybe change this to hands.

Reply:
Yeah if you lube it while its in your hands it drops to the ground. I think I'll just leave it like that for now because that makes sense even if its a little odd with the inspecting!